spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/SSLConnectionTests.java

```diff
@@ -82,6 +89,69 @@ public void testNoAlgNoProps() throws Exception {
 	}
 
 	@Test
+	public void testKSTS() throws Exception {
+		RabbitConnectionFactoryBean fb = new RabbitConnectionFactoryBean();
+		Log logger = spy(TestUtils.getPropertyValue(fb, "logger", Log.class));
+		given(logger.isDebugEnabled()).willReturn(true);
+		new DirectFieldAccessor(fb).setPropertyValue("logger", logger);
+		ConnectionFactory rabbitCf = spy(TestUtils.getPropertyValue(fb, "connectionFactory", ConnectionFactory.class));
+		new DirectFieldAccessor(fb).setPropertyValue("connectionFactory", rabbitCf);
```

Comment:
Well, that is copy/paste from existing tests, but this one doesn't use `rabbitCf` mock. 
So, we don't need to spy it